app/src/main/java/com/example/keybindhelper/MainActivity.kt

```diff
@@ -76,7 +76,7 @@ class MainActivity : AppCompatActivity() {
         // Inflate the menu; this adds items to the action bar if it is present.
         menuInflater.inflate(R.menu.main, menu)
         this.Menu=menu;
-        onMenuInit?.menuHasInitialized();
+        onMenuInit?.menuHasInitialized()
```

Comment:
Got rid of semicolons since they are not necessary in kotlin. Source: https://kotlinlang.org/docs/coding-conventions.html#horizontal-whitespace

"In Kotlin, semicolons are optional, and therefore line breaks are significant. The language design assumes Java-style braces, and you may encounter surprising behavior if you try to use a different formatting style."